### ISSUE TITLE
Incorrect parsing of `---` in code blocks?

### ISSUE THREAD

**Issue by frioux** [houseabsolute/Markdent#10]
I am not 100% sure of this but at the very least some markdown parsers parse this differently; The following (first quoted with spaces, then raw as a demonstration:)

    ```
    ---
    frew
    ---
    ```

(ie)

```
---
frew
---
```

Renders as:

```
<h2>```
</h2><h2>frew
</h2><p>```
</p>
```

in Markdent

**Comment by autarch** [comment]
Definitely a bug. It should be fairly easy to fix.

**Comment by autarch** [comment]
Fixed in v0.29. Thanks!
